src/runner/browser-env/vite/plugins/generate-index-html.ts

```diff
@@ -1,6 +1,7 @@
 import path from "node:path";
 import url from "node:url";
 import { builtinModules } from "node:module";
+import _ from "lodash";
```

Comment:
This plugin executes in nodejs env so there are no problems with import all methods from module.